### PR TITLE
fix image height and alert

### PR DESCRIPTION
**Pull request by kamiranoff**
FIx Image height in Cards
Fix Alert rendering issues and height on iphone with notch

### CODE REVIEW

File: src/common/GeneralAlert/index.tsx [hunk 1]
```diff
@@ -12,93 +13,97 @@ export enum NotificationMessages {
...
 
-const GeneralAlert: FC<Props> = ({ isExecuting, text, onAnimationFinish }) => {
-  const bounceVal = useRef(new Animated.Value(-100)).current;
-  const isThankYou = text === NotificationMessages.THANK_YOU;
-  const isError = text === NotificationMessages.ERROR;
+export interface GeneralAlertRef {
+  startAnimation: () => void;
+}
+
+const GeneralAlert = forwardRef<GeneralAlertRef, Props>(
```
Comment (heron2014) [inline src/common/GeneralAlert/index.tsx:24]: this is great!!

**Review by heron2014 — APPROVED** [review]
great work!

File: src/common/GeneralAlert/index.tsx [hunk 2]
```diff
@@ -12,93 +13,97 @@ export enum NotificationMessages {
...
-      }),
-    ]).start(() => {
-      if (onAnimationFinish) {
-        onAnimationFinish();
+    // The component instance will be extended
+    // with whatever you return from the callback passed
+    // as the second argument
+    useImperativeHandle(ref, () => ({
+      startAnimation() {
+        console.log('runAnimation');
```
Comment (kamiranoff) [inline src/common/GeneralAlert/index.tsx:43]: oops

Comment (heron2014) [reply src/common/GeneralAlert/index.tsx:43]: thats fine.. I will add sentry in next PR

File: src/common/GeneralAlert/index.tsx (hunk 1 above)
Comment (kamiranoff) [reply src/common/GeneralAlert/index.tsx:24]: yeah. shouldn't be abused. but in that specific case where you want to trigger a render even if the state doesn't change that should be ok

**Pull request merged**